app/web_conn.go

```diff
@@ -104,8 +105,17 @@ func (c *WebConn) WritePump() {
 				return
 			}
 
+			var msgBytes []byte
```

Comment:
Copy clone/copy the point to model.WebSocketEvent right here then set the sequence number, then send it ?  That way it's only copied once.